File: java/01-commands/tests/sourcecode/src/test/java/com/training/app/SumOfDigitsKata/NumberTestTemplate1.java
package com.training.app.SumOfDigitsKata;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class NumberTestTemplate1 {

    @Test
    public void given0_resultShouldBe0() {
        long num = 0;
        int expectedRes = 0;
        Number number = new Number(num);

        long res = number.digitalRoot();

        assertEquals(expectedRes, res);
    }

    @Test
    public void given10_resultShouldBe1() {
        long num = 10;
        int expectedRes = 1;
        Number number = new Number(num);

        long res = number.digitalRoot();

        assertEquals(expectedRes, res);
    }
}

File: dotnet/01-commands/tests/SumOfDigitsKata/NumberTestTemplate1.cs
using Xunit;

namespace SumOfDigitsKata
{
    public class NumberTestTemplate1
    {
        [Fact]
        public void Given0_ResultShouldBe0()
        {
            long num = 0;
            int expectedRes = 0;
            Number number = new Number(num);

            long res = number.DigitalRoot();

            Assert.Equal(expectedRes, res);
        }

        [Fact]
        public void Given10_ResultShouldBe1()
        {
            long num = 10;
            int expectedRes = 1;
            Number number = new Number(num);

            long res = number.DigitalRoot();

            Assert.Equal(expectedRes, res);
        }
    }
}